HPXMLtoOpenStudio/measure.rb

```diff
@@ -478,6 +478,21 @@ def self.set_defaults_and_globals(runner)
         sqft_by_gal = surface_area / act_vol # sqft/gal
         water_heating_system.standby_loss = (2.9721 * sqft_by_gal - 0.4732).round(3) # linear equation assuming a constant u, F/hr
       end
+      if (water_heating_system.water_heater_type == HPXML::WaterHeaterTypeStorage)
+        if water_heating_system.heating_capacity.nil?
+          water_heating_system.heating_capacity = Waterheater.get_default_heating_capacity(water_heating_system.fuel_type, @nbeds, @hpxml.water_heating_systems.size, @nbaths)
```

Comment:
This isn't correct -- note the units being returned from the method vs the units that HPXML uses. When you fix this, double-check the documentation too.

Reply:
😱 I fixed it.